### ISSUE TITLE
Week-3: Prepare project requirements.

### ISSUE THREAD

**Issue by enshkn** [enshkn/BOUN-SWE-573-SPRING-23#9]
**Why Priority is High:** Proper planning at the Requirement level will strengthen the likelihood that the whole project will be completed in a healthy way.

- Be focus on story, search, location, and time attributes.
- Be focus on resolution of time.

**Comment by enshkn** [comment]
The requirements are currently divided into 6 groups, these groups are Definitions, General, Profile, Memory, UI(Homepage), UI(Memories).
While determining the requirements, we tried to act with the minimum requirement that is considered necessary for the operation of the designed application.
The reason for this is to keep the system complexity at the lowest possible level at the beginning and to run the application first. At this point, care will be taken to use a modular architecture when coding the basic system designed in this form.

In the following stages, the requirements for additional features added to the application will be kept up to date with the requirements by adding these six headings and new headings to be opened if necessary.

After seeing that the system works, the required features will be added to the application designed and planned to be created modularly. 

The success of the application will not be measured by the quantity of features in the system. Instead, if it is not possible to remove even one of the features used in the application, the application will be considered successful. As simple as possible.

In other words, simplicity and modularity are accepted as success metrics.
The requirements can be accessible from this [link](https://github.com/enshkn/BOUN-SWE-573-SPRING-23/wiki/Assigments-&-Researches#draft-requirements-1)
